src/libAtomVM/opcodesswitch.h

```diff
@@ -6460,7 +6460,7 @@ static bool maybe_call_native(Context *ctx, AtomString module_name, AtomString f
                                 if (size != ALL_ATOM) {
                                     VERIFY_IS_INTEGER(size, "bs_create_bin/6");
                                     size_value = term_to_int(size);
-                                    if (size_value > binary_size) {
+                                    if ((size_t) size_value > binary_size) {
```

Comment:
here the same, let's try avoid casts.

Reply:
Fixed, though a cast is still used (after a check)